tests/conftest.py

```diff
@@ -1,33 +1,110 @@
 import json
 from datetime import datetime, timezone
+from unittest.mock import MagicMock, patch
 
+import boto3
 import pytest
+from botocore.stub import Stubber
+
+# Usage and cost data fixtures for test_cost.py
 
 
 @pytest.fixture(scope="function")
 def input_data_usage():
-    with open("tests/test_data_usage.json") as f:
+    with open("tests/data/test_data_usage.json") as f:
         data = json.load(f)
     return data
 
 
 @pytest.fixture(scope="function")
 def input_data_cost():
-    with open("tests/test_data_cost.json") as f:
+    with open("tests/data/test_data_cost.json") as f:
         data = json.load(f)
     return data
 
 
 @pytest.fixture(scope="function")
 def output_data_hub():
-    with open("tests/test_output_hub.json") as f:
+    with open("tests/data/test_output_hub.json") as f:
         data = json.load(f)
     return data
 
 
 @pytest.fixture(scope="function")
 def output_data_component():
-    with open("tests/test_output_component.json") as f:
+    with open("tests/data/test_output_component.json") as f:
+        data = json.load(f)
+    return data
+
+
+# Usage and cost data for test_integration.py
+
+
+@pytest.fixture(autouse=True)
+def env_vars(monkeypatch):
+    """
+    Set environment variables for testing.
+    """
+    monkeypatch.setenv("CLUSTER_NAME", "test-cluster")
+    monkeypatch.setenv("AWS_EC2_METADATA_DISABLED", "true")
+    # Fake AWS credentials for boto3 client
+    monkeypatch.setenv("AWS_ACCESS_KEY_ID", "fake-key")
+    monkeypatch.setenv("AWS_SECRET_ACCESS_KEY", "fake-secret")
+    monkeypatch.setenv("AWS_SESSION_TOKEN", "fake-token")
```

Comment:
not sure if I understand correctly, but may be we only need `os.environ["CLUSTER_NAME"] = "test-cluster"` here?

Reply:
`os.environ["CLUSTER_NAME"] = "test-cluster"` needs to be set before the test collection begins -- so may be at the module level instead of inside a fixture